habitat-lab/habitat/utils/gym_definitions.py

```diff
@@ -64,24 +84,7 @@ def _make_habitat_gym_env(
 
     config = habitat.get_config(cfg_file_path, overrides=override_options)
     if use_render_mode:
-        with habitat.config.read_write(config):
-            sim_config = config.habitat.simulator
-            default_agent_name = sim_config.agents_order[
-                sim_config.default_agent_id
-            ]
-            default_agent = sim_config.agents[default_agent_name]
-            if len(sim_config.agents) == 1:
-                default_agent.sim_sensors.update(
-                    {"third_rgb_sensor": ThirdRGBSensorConfig()}
-                )
-            else:
-                default_agent.sim_sensors.update(
-                    {
-                        "default_agent_third_rgb_sensor": ThirdRGBSensorConfig(
-                            uuid="default_robot_third_rgb"
-                        )
-                    }
-                )
+        add_sim_sensor_to_config(config, ThirdRGBSensorConfig())
```

Comment:
Shouldn't this get the sensors from `config.habitat_baselines.eval.extra_sim_sensors` instead?

Reply:
When creating the gym environment, the habitat baselines config won't be available, so I added a generic `ThirdRGBSensorConfig` instead.

Reply:
Good point, this makes sense.